### PR TITLE
backport: v20.0.3 backports and release 

### PR DESCRIPTION
**Pull request by ogabrielides**
## Issue being fixed or feature implemented
v20.0.3 backports and release

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

**Comment by UdjinM6** [comment]
Looks good so far, waiting for #5784

File: doc/release-notes.md [hunk 1]
```diff
@@ -37,18 +37,18 @@ reindex or re-sync the whole chain.
 
 ## Masternode fix
 
-A problem has been fixed in the old quorum data cleanup mechanism. It was slowing down masternodes during DKG sessions and causing them to get PoSe scored.
+The momory usage during the old quorum data cleanup mechanism was reduced.
```
Comment (UdjinM6) [inline doc/release-notes.md:40]:
```suggestion
The memory usage during the old quorum data cleanup mechanism was reduced.
```

File: doc/release-notes-5774.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+RPC changes
+-----------
+
+In `getspecialtxes` `instantlock` and `chainlock` fields were always `false`. They should show actual values now.
```
Comment (UdjinM6) [inline doc/release-notes-5774.md:4]: should add this to release notes and drop this file

File: doc/release-notes.md [hunk 3]
```diff
@@ -37,18 +37,22 @@ reindex or re-sync the whole chain.
...
 
-## Testnet Crash
+## Wallet fix
 
-A fix has been implemented for the reported crash that could occur when upgrading from v19.x to v20.0.0 after v20 activation without re-indexing.
+A fix has been implemented for the reported decryption of wallets.
+
+## RPC changes
+
+In `getspecialtxes` `instantlock` and `chainlock` fields are now reflecting actual values now.
```
Comment (UdjinM6) [inline doc/release-notes.md:48]: such text much `now`
```suggestion
In `getspecialtxes` `instantlock` and `chainlock` fields are reflecting actual values now.
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Comment by ogabrielides** [comment]
So I guess we want to include #5784 as well?

**Comment by PastaPastaPasta** [comment]
Yes; Looks like it

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit; I pushed a no-diff rebase so that these commits were gpg signed.

**Comment by PastaPastaPasta** [comment]
Guix Automation has began to build this PR tagged as v20.0.3-devpr5785.3192d8c9. A new comment will be made when the image is pushed.

**Pull request merged**